helmfile/values/workload-cluster-networkpolicy.yaml.gotmpl

```diff
@@ -10,6 +10,8 @@ global:
     ips: {{- toYaml .Values.networkPolicies.global.scIngress.ips | nindent 6 }}
   ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
   externalLoadBalancer: {{ .Values.networkPolicies.global.externalLoadBalancer }}
+  trivy:
+    ips: {{- toYaml .Values.networkPolicies.global.trivy.ips | nindent 6 }}
```

Comment:
From what I can tell this isn't used in `networkpolicy-workload` anymore. But if it is I guess you'll want to add the port as well

Reply:
It is not used. i have removed it